lizmap/definitions/filter_by_polygon.py

```diff
@@ -69,6 +75,16 @@ def __init__(self):
             'tooltip': tr('The spatial relationship to use when filtering data')
         }
 
+        self._layer_config['use_centroid'] = {
+            'type': InputType.CheckBox,
+            'header': tr('Centroid'),
+            'default': False,
+            'tooltip': tr(
+                'If the tool use the centroid of the geometry or the full geometry. It\'s quicker to use centroid. '
+                'For PostgreSQL layer, an index on the centroid is required.'
```

Comment:
```suggestion
                'If the tool must use the centroid of the geometry or the full geometry. It\'s quicker to use the centroid. '
                'For PostgreSQL layers, an index on the centroid is required.'
```

Reply:
I fixed all suggestions except this one.
As we are in a form, configuring a single layer, I wanted to keep it singular. I just edited with "For **a** PostgreSQL layer,..."

Hope it's alright ?

Reply:
ok for me, thanks